ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/ChainSel.hs

```diff
@@ -782,7 +787,6 @@ chainSelectionForBlock cdb@CDB{..} blockCache hdr punish =
           mkTraceEvent events (mkNewTipInfo newLedger) curChain newChain
         whenJust (strictMaybeToMaybe prevTentativeHeader) $ traceWith $
           PipeliningEvent . OutdatedTentativeHeader >$< addBlockTracer
-        traceWith cdbTraceLedger newLedger
```

Comment:
No point in tracing always with the null tracer (plus tracing a whole LedgerDB is an overkill :thinking: )